tools/core.mk

```diff
@@ -0,0 +1,106 @@
+
+rfcdiff = rfcdiff --width 78 --stdout
+reduction  = $(tools)/clean-for-DTD.xslt
+bap = $(tools)/bap
+bd  = build
+
+draftname = draft-ietf-httpbis
+
+AUTH        = $(draftname)-auth
+CACHE       = $(draftname)-cache
+CONDITIONAL = $(draftname)-conditional
+MESSAGING   = $(draftname)-messaging
+RANGE       = $(draftname)-range
+SEMANTICS   = $(draftname)-semantics
+
+CORE_XML = $(MESSAGING).xml $(SEMANTICS).xml
+TARGETS_ABNF= $(addprefix $(bd)/,$(CORE_XML:.xml=.abnf))
+TARGETS_ABNFAPPENDIX= $(TARGETS_ABNF:.abnf=.abnf-appendix)
+TARGETS_PARSEDABNF= $(TARGETS_ABNF:.abnf=.parsed-abnf)
+
+ABNF =    $(TARGETS_ABNF) \
```

Comment:
It looks like these files are still being checked-in to the repo.  That seems problematic.